src/Extension/FluentSiteTreeExtension.php

```diff
@@ -158,33 +158,6 @@ protected function updateLink(&$link, &$action, &$relativeLink)
         $link = Controller::join_links($domain->Link(), $link);
     }
 
-    /**
-     * Check whether the current page is exists in the current locale.
-     *
-     * If it is invisible then we add a class to show it slightly greyed out in the site tree.
-     *
-     * @param array $flags
-     */
-    protected function updateStatusFlags(&$flags)
```

Comment:
Nothing here is specific to SiteTree anymore. This is now split up between `FluentExtension` and `FluentVersionedExtension` along with the methods called from here.